.github/workflows/srtool.yml

```diff
@@ -0,0 +1,58 @@
+name: Srtool build
+
+on: push
```

Comment:
I don't think we want to run the workflow on every push?

Reply:
@s3krit Can we run it on a release? E.g. on certain branch names or tags?

Reply:
Yep dead simple - if you know the pattern of what cumulus release tags look like, you can match like so: https://github.com/paritytech/polkadot/blob/master/.github/workflows/publish-draft-release.yml#L7